urbs.py

```diff
@@ -1533,22 +1533,25 @@ def result_figures(prob, figure_basename, plot_title_prefix=None, periods={}):
     # default to all timesteps if no
     if not periods:
         periods = {'all': sorted(get_entity(prob, 'tm').index)}
```

Comment:
I would rather fix this here in line 1535. The other check is quite convoluted. It is rather a runme.py deficiency right now to set set 't' to {5000,...} and then desire a plot of that timestep as well...
Generally, this initialization time step could probably be handled much less user intrusive. I'm still figuring out the best why to hide this 0'th step completely.